### PR TITLE
docs(ballot-interpreter): add a high-level overview

### PR DESCRIPTION
**Pull request by eventualbuddha**
I'm sure I missed something. Please comment if I did or if something isn't clear!

### CODE REVIEW

File: libs/ballot-interpreter/README.md [hunk 1]
```diff
@@ -96,6 +96,114 @@ To update the saved results for a benchmark, run:
...
+
+1. **Allow for some missing timing marks.** We infer missing timing marks based
+   on the spacing between known timing marks. This is especially important for
+   bottom timing marks, which may be used to encode the ballot metadata. We
+   can't allow too many missing timing marks, however, because the inference
+   process is not perfect and may introduce errors.
+2. **Require small rotation and/or skew if we've inferred any timing marks.** If
+   we didn't infer any timing marks we can be more lenient, but if we did we
+   need to be more strict about rotation and skew because they can cause the
+   inferred timing marks to be incorrect. Note that this projects uses
```
Comment (benadida) [inline libs/ballot-interpreter/README.md:162]:
```suggestion
   inferred timing marks to be incorrect. Note that this project uses
```

**Review by benadida — APPROVED** [review]
I love this, thank you for adding it.

**Review by arsalansufi — COMMENTED** [review]
:tada: Thanks for writing this up!

File: libs/ballot-interpreter/README.md [hunk 2]
```diff
@@ -96,6 +96,114 @@ To update the saved results for a benchmark, run:
...
+   need to be more strict about rotation and skew because they can cause the
+   inferred timing marks to be incorrect. Note that this project uses
+   "rotation" to refer to an angle by which the entire ballot is rotated,
+   preserving the equal distance of points in the grid from each other. "Skew"
+   refers to a distortion of the grid, where the distance between points in the
+   grid is not preserved, and is caused by different parts of the ballot being
+   scanned at different speeds.
+3. **Require that the number of inferred timing marks on a side matches the
+   expected number.** And that the left/right and top/bottom marks have the same
+   count.
```
Comment (arsalansufi) [inline libs/ballot-interpreter/README.md:170]: > And that the left/right and top/bottom marks have the same count

I assume the one exception to top/bottom marks having the same count is if we're encoding metadata using the bottom timing marks, is that correct?

Comment (eventualbuddha) [reply libs/ballot-interpreter/README.md:170]: Yes, that's correct. Do you think it's worth clarifying here?

Comment (arsalansufi) [reply libs/ballot-interpreter/README.md:170]: I think it's worth a quick mention!

Comment (eventualbuddha) [reply libs/ballot-interpreter/README.md:170]: https://github.com/votingworks/vxsuite/pull/5205

**Pull request merged**